pkg/machines/components/nicEdit.jsx

```diff
@@ -56,14 +56,17 @@ class EditNICModal extends React.Component {
         let defaultNetworkSource;
         let currentSource;
         let availableSources = [];
+
         if (props.network.type === "network") {
-            availableSources = props.networks.map(network => network.name);
```

Comment:
Now that networks prop is not used anywhere, so we could just not pass it to this component at all.

Reply:
It still gets passed on to `NetworkTypeAndSourceRow`, no?

Reply:
Yeah and in that component it's not used anywhere

Reply:
I will create another PR for that